docs/builtins.md

```diff
@@ -511,75 +511,75 @@ line 4 column 5: scale must be positive
 The `panic` function takes a single argument, which is the error message
 that the program will print before it terminates with exit status 1.
 
-### `assert`
+### `test`
 
-`assert` is used to check if a condition is true or two values – `want` and
+`test` is used to check if a condition is true or two values – `want` and
 `got` – are the same. If the condition is not true or the two values are not
-the same, the program will print the failed assertion and terminate with exit
+the same, the program will print the failed test and terminate with exit
 status 1.
 
-`assert` optionally takes a message or a formatted message with arguments to
-print along with the failed assertion as a third and following arguments:
+`test` optionally takes a message or a formatted message with arguments to
```

Comment:
I think "formatted message" is wrong here. It should say something like "format string". The text below is more specific about sprintf, etc so I think this general term is fine.